app/styles/_tables.less

```diff
@@ -123,6 +123,7 @@
            /* Behave like a row */
           border: none;
           border-bottom: 1px solid lighten(@table-border-color, 5%);
+          min-height: 38px; // prevent empty td collapsing
```

Comment:
This seems like a magic number. Is there any existing variable that makes sense for min-height?